### PR TITLE
Support LLVM 14

### PR DESCRIPTION
**Pull request by RyanGlScott**
All of the changes needed to support LLVM 14 involve new metadata fields.

### CODE REVIEW

**Comment by langston-barrett** [comment]
I see some `btf_tag` content, will this PR close https://github.com/GaloisInc/llvm-pretty-bc-parser/issues/188?

**Review by langston-barrett — APPROVED** [review]
Love it!

File: README.md [hunk 1]
```diff
@@ -64,3 +66,5 @@ the developers' documentation for more details and a rationale:
 [fuzz-workflow]: https://github.com/GaloisInc/llvm-pretty-bc-parser/blob/master/.github/workflows/llvm-quick-fuzz.yml
 [llvm13]: https://github.com/GaloisInc/llvm-pretty-bc-parser/issues?q=is%3Aopen+is%3Aissue+label%3Allvm%2F13.0
 [llvm14]: https://github.com/GaloisInc/llvm-pretty-bc-parser/issues?q=is%3Aopen+is%3Aissue+label%3Allvm%2F14.0
+[llvm15]: https://github.com/GaloisInc/llvm-pretty-bc-parser/issues?q=is%3Aopen+is%3Aissue+label%3Allvm%2F15.0
+[llvm16]: https://github.com/GaloisInc/llvm-pretty-bc-parser/issues?q=is%3Aopen+is%3Aissue+label%3Allvm%2F16.0
```
Comment (kquick) [inline README.md:70]: BTW, I really like providing these links to issue tags!

**Review by kquick — APPROVED** [review]

**Comment by RyanGlScott** [comment]
Successful `llvm-quick-fuzz` job including LLVM 14: https://github.com/GaloisInc/llvm-pretty-bc-parser/actions/runs/4656935372

**Pull request merged**